R/mod-upload-documentation.R

```diff
@@ -1,7 +1,7 @@
 #' UI function for the upload_documentation module
 #' @param id the module id
 #' @return html ui for the module
-upload_documents_ui <- function(id) {
+upload_documents_ui <- function(id, study_link_human, study_link_animal) {
```

Comment:
Could you please document the new parameters in the roxygen comments above the function and then run `devtools::document()`?